storage_service/storage_service/settings/base.py

```diff
@@ -12,6 +12,12 @@
 from django.core.exceptions import ImproperlyConfigured
 from django.utils.translation import ugettext_lazy as _
 
+try:
+    import ldap
+    from django_auth_ldap import config as ldap_config
+except ImportError:
+    ldap, ldap_config = None, None
```

Comment:
Torn here if handling the case where ldap is not installed is worthwhile or not, but I thought it might ease some pain for people who don't care about LDAP and don't want to install the headers. It may be worth making an ldap specific requirements file if we go this route, though.